c2corg_ui/templates/helpers/view.html

```diff
@@ -89,10 +89,16 @@ <h4 translate>No maps associated</h4>
 
 <%def name="show_areas(document)">\
   % if document.get('areas'):
-  <% areas = document.get('areas') %>\
+    <%
+        areas = document.get('areas')
```

Comment:
That's a 4-space indent, as in python and in the mako documentation:
http://docs.makotemplates.org/en/latest/syntax.html#python-blocks

I know we have not been very consistent and that one can find such python blocks indented or not in our templates. I prefer to stick to the doc recommendation.